src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -548,7 +545,18 @@ protected DynamoModel(IStartConfiguration config)
 
             StartBackupFilesTimer();
 
-            TraceReconciliationProcessor = this;
+            TraceReconciliationProcessor = this; 
+            
+            foreach (var ext in ExtensionManager.Extensions)
+            {
+                ext.Ready(new ReadyParams());
```

Comment:
If the `Extension` throws an exception, we should catch it here and log the exception message.